### PR TITLE
QC

### PR DESCRIPTION
**Pull request by EmmaRenauld**

### CODE REVIEW

File: source/our_tools/other_pipelines.rst [hunk 1]
```diff
@@ -3,13 +3,33 @@
...
 
 QA/ QC  (Quality assurance, Quality check)
 ------------------------------------------
 
-We have a QA pipeline which creates a summary of the quality of your data, you can ask Guillaume if you are interested. ToDo
+We have a QC pipeline which creates a summary of the quality of your data.
+
+*How to install it?*
+
+    The easiest is to use it as a nextflow with singularity (see :ref:`ref_nextflow` for more information.) First download the singularity image: On this page, https://github.com/GuillaumeTh/dmriqcpy/actions, click on the latest version, then under "Artifacts", click on the singularity to start the download. You should take a note somewhere to remember which version you downloaded. Then, download the nextflow:
```
Comment (GuillaumeTh) [inline source/our_tools/other_pipelines.rst:18]: Now the singularity is available in release tab: https://github.com/GuillaumeTh/dmriqcpy/releases

File: source/our_tools/other_pipelines.rst [hunk 2]
```diff
@@ -3,13 +3,33 @@
...
+
+    .. code-block:: bash
+
+        git clone https://github.com/GuillaumeTh/dmriqc-flow.git
+
+*How to use it?*
+
+    .. code-block:: bash
+
+        tractoflow_results=YOUR_PATH
```
Comment (GuillaumeTh) [inline source/our_tools/other_pipelines.rst:28]: I don't like to use variables for this use case. Just the command like this is ok:   
nextflow run PATH_TO_DMRIQC_FLOW/main.nf --root TRACTOFLOW_RESULTS -with-singularity PATH_TO_THE_DMRIQC_SINGULARITY -resume

**Comment by EmmaRenauld** [comment]
Thanks. I am testing it right now.
Do you have a list of things you typically check visually to see the results?

**Comment by EmmaRenauld** [comment]
Tested, it worked. Everything good, @GuillaumeTh ?

**Pull request merged**